accounts/accounts_cli.go

```diff
@@ -0,0 +1,328 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/codegangsta/cli"
+	"github.com/howeyc/gopass"
+)
+
+// Represents the user running the audit on the accounts that they have access to via permissons
+type admin struct {
+	host   string
+	client *http.Client
+	User   struct {
+		Id    string `json:"userid"`
+		Name  string `json:"username"`
+		Token string `json:"-"`
+	} `json:"administrator"`
+	Accounts []*Account `json:"administeredAccounts"`
+}
+
+// Account details that we will audit
+type Account struct {
+	Id      string `json:"userid"`
+	Profile struct {
+		FullName string `json:"fullName"`
+		Patient  struct {
+			Bday string `json:"birthday"`
+			Dday string `json:"diagnosisDate"`
+		} `json:"patient"`
+	} `json:"patientProfile"`
+	Perms      interface{} `json:"permissons"`
+	LastUpload string      `json:"lastupload"`
```

Comment:
yeah we have a mixture at the moment including underscores. In this instance I have just matched it what is being returned so that I could build the report.

I am keen that we settle on camel case for JSON - we will just need to adjust across the board to do this.